state/carmen.go

```diff
@@ -324,6 +326,14 @@ func (s *carmenStateDB) GetLogs(common.Hash, uint64, common.Hash) []*types.Log {
 	return res
 }
 
+func (s *carmenStateDB) PointCache() *utils.PointCache {
+	panic("PointCache not implemented")
+}
+
+func (s *carmenStateDB) Witness() *stateless.Witness {
+	return nil
```

Comment:
I think we should `panic` here aswell, this could potentionally misslead to an error.

Reply:
ic thanks for the explanation